tests/sentry/api/endpoints/test_team_members.py

```diff
@@ -46,3 +46,12 @@ def test_team_members_list_does_not_include_invite_requests(self):
         assert len(response.data) == 2
         assert response.data[0]["id"] == str(self.team_member.id)
         assert response.data[1]["id"] == str(pending_invite.id)
+
+    def test_team_members_list_includes_roles(self):
```

Comment:
Do we have tests for the other queryset filters?

Reply:
ie. `is_active` / `is_null`

I guess not really a part of this PR, but just noticed that was touched